bin/es

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+declare ES_PROTOCOL="http"
+declare ES_HOST="localhost"
+declare -i ES_PORT=9200
+declare ES_INDEX="statements"
+declare ES_URL="${ES_PROTOCOL}://${ES_HOST}:${ES_PORT}"
+
+
+function usage(){
+    declare -i exit_code="${1:-0}"
+
+    echo "Usage: bin/es COMMAND
+
+COMMAND:
+
+  index       create elasticsearch index and fake documents
+    "
+
+    exit "${exit_code}"
+}
+
+
+function index(){
+
+  local data
+  local index="${1:-${ES_INDEX}}"
+
+  # timestamp in ms (as required by elasticsearch)
+  local now
+  now=$(echo "$(date +%s.%N) * 1000" | bc)
+
+  for i in $(seq 100); do
+    uuid=$(uuidgen)
+    # Simulate one event per minute
+    at=$(echo "${now} - (60 * 1000 * ${i})" | bc)
```

Comment:
Another alternative here, you can remove the `$now` variable and use the following :

```suggestion
    at=$(date -d "now -${i}min" +"%s.%N000")
```

The only difference is that `now` will not be a fixed value and will be relative to the time when the command is executed. In your use case i don't think it's a problem.

Reply:
Wow didn't know about this pattern, thanks.